src/utils/file.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, output_dir):
         self.manual_dir = output_dir.joinpath("Manual")
         self.errors_dir = self.manual_dir.joinpath("ErrorFiles")
         self.multi_marked_dir = self.manual_dir.joinpath("MultiMarkedFiles")
+        self.eval_dir = output_dir.joinpath("Evaluations")
```

Comment:
Let's rename it to bit more verbose
```suggestion
        self.evaluations_dir = output_dir.joinpath("Evaluations")
```